scripts/deploy-multidev.sh

```diff
@@ -108,6 +108,13 @@ if [ "$CIRCLE_BRANCH" != "master" ] && [ "$CIRCLE_BRANCH" != "dev" ] && [ "$CIRC
     export avoid_redirect="window.location.hostname == '$normalize_branch-static-docs.pantheonsite.io' ||"
     sed -i '9i\'"      ${avoid_redirect}"'\' source/_views/default.html
     sed -i '9i\'"      ${avoid_redirect}"'\' source/_views/taxon.html
+    export avoid_redirect2="window.location.hostname == '$normalize_branch-static-docs.pantheon.io' ||"
+    sed -i '9i\'"      ${avoid_redirect2}"'\' source/_views/default.html
+    sed -i '9i\'"      ${avoid_redirect2}"'\' source/_views/taxon.html
+    export avoid_redirect3="window.location.hostname == '$normalize_branch-static-docs.gotpantheon.com' ||"
+    sed -i '9i\'"      ${avoid_redirect3}"'\' source/_views/default.html
+    sed -i '9i\'"      ${avoid_redirect3}"'\' source/_views/taxon.html
+
```

Comment:
@nataliejeremy @bmackinney I added rewrites to legacy platform domains to the multidev deployment script - this way if legacy domains are provisioned for the multidev they will at least be accessible. If clicking on the URL from slack or the comment on github opens a page with a DNS error, the doc is more than likely on another platform domain such as: http://1452migrat-static-docs.gotpantheon.com/docs/migrate/